docs/_dapp/tracing.md

```diff
@@ -208,9 +208,3 @@ transaction touches (balances, code, nonce etc). It should be mentioned that the
 reproducer is strictly guaranteed to be totally exact with regards to gascosts incurred by
 the outer transaction, as evmlab does not fully calculate the gascosts for nonzero data
 etc, but is usually sufficient to analyze contracts and events.
-
```

Comment:
It doesn't hurt to leave those links here, so that people don't have to read the entire text to find a link that they are looking for.

Reply:
Do you mean for people reading the raw markdown file? Asking because these links are not actually displayed in the rendered web page